pkgs/servers/home-assistant/custom-themes/material-you-theme/package.nix

```
@@ -0,0 +1,36 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "material-you-theme";
+  version = "5.0.1";
+
+  src = fetchFromGitHub {
+    owner = "Nerwyn";
+    repo = "material-you-theme";
+    tag = version;
+    hash = "sha256-xJXhvKwp/l08/ZWi3OcGPmCdsUiMjBDwrKz5OIpD2t8=";
+  };
+
+  dontBuild = true;
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir $out
+    cp themes/material_you.yaml $out
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
```

```suggestion
  meta = {
```
https://github.com/NixOS/nixpkgs/issues/208242